src/helpers.js

```diff
@@ -120,6 +124,17 @@ export async function afterCLA(app, claSignatureInfo) {
   console.log(`Processing CLA for ${githubUsername ? `user: ${githubUsername}` : 'unknown user'} in org/account: ${org}`);
   
   try {
+    const query = `org:${org} is:pr is:open author:${githubUsername}`;
+    // GitHub Docs for octokit.rest.search - https://github.com/octokit/plugin-rest-endpoint-methods.js/tree/main/docs/search
+    const { data: { items: prs } } = await app.octokit.rest.search.issuesAndPullRequests({
+      q: query,
+      sort: "updated",
+      order: "desc"
+    });
+
+    const filteredPrs = prs.filter(pr => pr.user.login === githubUsername);
+    console.log(`Found ${filteredPrs.length} open PRs for ${githubUsername} in ${org}:`, filteredPrs.map(pr => pr.number).join(', '));
+
     for await (const { installation } of app.eachInstallation.iterator()) {
```

Comment:
Is this really needed? 

1. Why not use the app.octokit instance for removeLabel call?
2. Is installationId (and hence auth scope) different for different repo of the same org?
3. Can we have a more unified way to get the correct authenticated octokit instance? e.g. getAuthenticatedInstance(repo, type)